pkg/asset/installconfig/openstack/openstack.go

```diff
@@ -67,37 +73,41 @@ func Platform() (*openstack.Platform, error) {
 			}),
 		},
 	}, &extNet)
-	if err != nil {
-		return nil, err
+	if extNet == noExtNet {
+		extNet = ""
 	}
-
-	floatingIPNames, err := validValuesFetcher.GetFloatingIPNames(cloud, extNet)
 	if err != nil {
```

Comment:
nit: this error check can be move to above line 76

Reply:
don't think so, at line 76, extNet could = "<None>" (noExtNet), which would cause the call to fail.